packages/nimbuild-corejs/lib/utils/normalizeUnreleased.js

```diff
@@ -0,0 +1,28 @@
+const browserslist = require('browserslist');
+
+/**
```

Comment:
This function needed to be created because `browserslist` does not support a query that accommodates "Is this a valid version, including if it's out of bounds of my currently defined data?"

FWIW, another user had a similar request for queries for unreleased versions, but for nimbuild-corejs that came very close to implementing this: https://github.com/browserslist/browserslist/issues/163.  In the issue they called for future releases potentially, but instead settled on `unreleased` which was limited.

I think this sort of functionality could be accommodated through an option, like:

```
browserslist('chrome 140', { includeFutureReleases: true });
```

Then `core-js-compat` and `nimbuild-corejs` would both need to update their implementations of browserslist to use this option.